server/src/main/java/com/openelements/benchscape/server/store/endpoints/MeasurementEndpoint.java

```diff
@@ -123,4 +119,10 @@ List<Measurement> findByQuery(@RequestParam final String benchmarkId,
             return measurements;
         }
     }
+
+    @GetMapping("/{benchmarkId}/branches")
+    public List<String> getBranchesForBenchmark(@PathVariable String benchmarkId) {
```

Comment:
lets rename it in getGitBranchesForBenchmark since I needed a second to get what is meant here.